### PR TITLE
post: typescript part 1

### PR DESCRIPTION
**Pull request by cexbrayat**
First half of the TS part.
Should be published on Tuesday!

### CODE REVIEW

File: _posts/2015-11-10-typescript-part-1.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+description: "Angular 2 is not that far now. It has been designed to leverage a lot of new wonderful things in Web development, like TypeScript. Let's have a look!"
+---
+
+TypeScript has been around since 2012, and is a superset of JavaScript, adding a few things to ES5. The more important one is the type system, giving TypeScript its name. From version 1.5, released in 2015, the library is trying to be a superset of ES6, including all the shiny features we saw in the last blog posts, and a few new things as well, like decorators.
+Writing TypeScript feels very much like writing JavaScript. By convention, TypeScript files are named with a `.ts` extension, and they will need to be compiled to standard JavaScript, usually at build time, using the TypeScript compiler. The generated code is very readable.
+
+    npm install -g typescript
+    tsc test.ts
+
+But let's start by the beginning.
```
Comment (clacote) [inline _posts/2015-11-10-typescript-part-1.md]: "start **with**" plutôt, non?

**Comment by clacote** [comment]
No link in the beginning to the previous post that introduces this part?
Otherwise : :+1:

**Comment by cexbrayat** [comment]
@clacote new commit with your feedback

**Comment by clacote** [comment]
👍

**Comment by cexbrayat** [comment]
@clacote merged, ready for your community management magic

**Pull request merged**